cmd/thetool/links.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"sync"
+	"time"
+)
+
+var (
+	parallel *int
+	ignore   *bool
+)
+
+func checkLink(wg *sync.WaitGroup, in chan string, out chan error) {
+	defer wg.Done()
+	client := http.Client{
+		Timeout: time.Second,
+	}
+	for lnk := range in {
+		u, err := url.Parse(lnk)
+		if err != nil {
+			out <- fmt.Errorf("parse %q failed with err %w", lnk, err)
+			continue
+		}
+
+		req, err := http.NewRequest("GET", u.String(), nil)
+		if err != nil {
+			out <- fmt.Errorf("create request for %q failed with err %w", lnk, err)
+			continue
+		}
+
+		rsp, err := client.Do(req)
+		if err != nil {
+			out <- fmt.Errorf("fetch %q failed with err %w", lnk, err)
+			continue
+		}
+
+		if rsp.StatusCode != http.StatusOK {
+			out <- fmt.Errorf("fetch %q failed with status code %d", lnk, rsp.StatusCode)
+			continue
+		}
+	}
+}
+
+func validateLinks(_ *command, fl *File) error {
+	if *parallel < 1 {
+		*parallel = 3
+	}
+
+	// Who cares abou a large channel here :D
```

Comment:
`Who cares abou` -> `Who cares about`. `t` is missing :D

Reply:
good catch. also the comment wasn't necessary anymore :)